apps/finance/app/src/components/Balances.js

```diff
@@ -54,6 +54,23 @@ class Balances extends React.Component {
       }
     )
 
+    if (!balanceItems.length) {
+      return (
+        <Box heading="Token Balances">
+          <div
+            css={`
+              margin: ${5 * GU}px auto;
+              text-align: center;
+              ${textStyle('body1')};
+              color: ${theme.content};
+            `}
+          >
+            No token balances yet.
```

Comment:
cc @dizzypaty This would also be shown if someone had transferred out all of their tokens (e.g. they transferred 1 ETH in, and then transferred 1 ETH out).

Perhaps just cutting off the "yet" bit (so it becomes `No token balances`) is good enough for now to not be confusing in that case, and we can decide how to better show this case later.